packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -217,3 +233,34 @@ export class CloneRepository {
 
   private constructor(public readonly repositoryUrl: string, public readonly pathComponent: string) {}
 }
+
+/**
+ * The class for different types of owners
+ */
+export class Owner {
+  /**
+   * import from Owner Iuser
+   *
```

Comment:
Don't forget to include what restrictions there are on this user. For example, I recall reading something in the documentation to say that the user needed to have Cloud9Administrator permissions, or something. Would be good to include here, and a reference to the documentation page.